lte/gateway/c/session_manager/CMakeLists.txt

```diff
@@ -199,10 +199,12 @@ add_library(SESSION_MANAGER
     ${PROTO_SRCS}
     ${PROTO_HDRS})
 
+find_package(MAGMA_CONFIG REQUIRED)
```

Comment:
Not sure if this is really required, MAGMA_CONFIG was already used before the change.